doc/how-to/manage-infrastructure.md

```diff
@@ -0,0 +1,119 @@
+# Manage Infrastructure
+
+The service is hosted on the [MoJ Cloud
+Platform](https://user-guide.cloud-platform.service.justice.gov.uk/#getting-started).
+It's a platform where we can host our applications and interact with them
+through Kubernetes. This requires our applications to all be Dockerised.
+
+Each repository will have a `helm_deploy` directory that specifies configuration
+for this service in each of the environments.
+
+When this app deploys via merging to `main`, CircleCI will automatically
+propagate those changes to the cluster. This happens through the
+`hmpps/deploy_env` job that is provided by the [MoJ CircleCI
+Orb](https://github.com/ministryofjustice/hmpps-circleci-orb).
+
+Each environment will correspond to a Cloud Platform 'namespace'. The namespace
+is an isolated cluster. We can use the [Cloud Platform Environments
+repository](https://github.com/ministryofjustice/cloud-platform-environments/tree/main/namespaces/live.cloud-platform.service.justice.gov.uk)
+to define our backing services, certificates etc.
+
+## Prerequisites
```

Comment:
Prerequisites for what? Connecting to Kubernetes? Maybe this section should be `Connecting to Kubernetes`, with the first two points as prerequisites and the last as a link to instructions? (Unless you want to have a prerequisites section on both docs)